src/js/hooks/outboundImport/useOutboundImportForm.js

```diff
@@ -155,7 +155,15 @@ const useOutboundImportForm = ({ next }) => {
         itemsInOrder: tableDataGrouped.itemsInOrder,
       });
       setErrorsData({ errors: {}, validateStatus: validationResponse.status });
+      next();
     } catch (e) {
+      if (e.response.status !== HttpStatusCode.BadRequest) {
```

Comment:
@drodzewicz it is not caught, because for the full outbound import we took control over error handling, and we use `apiClientCustomResponseHandler` instead of `apiClient`.